ROMFS/px4fmu_common/init.d-posix/rc.replay

```diff
@@ -12,6 +12,7 @@ EOF
 uorb start
 param set SDLOG_DIRS_MAX 7
 
+replay tryapplyparams
```

Comment:
Can you add a short comment, like: # apply all params before ekf starts, as some params cannot be changed after startup

Reply:
okay, done